hq/app/model/models.scala

```diff
@@ -261,7 +263,16 @@ trait IAMAlert {
   def username: String
   def tags: List[Tag]
 }
+case class IAMAlertTargetGroup(targets: List[Target], outdatedKeysUsers: Seq[UserWithOutdatedKeys], noMfaUsers: Seq[UserNoMfa])
+
 case class UserWithOutdatedKeys(username: String, key1LastRotation: Option[DateTime], key2LastRotation: Option[DateTime], userLastActiveDay: Option[Long], tags: List[Tag]) extends IAMAlert
 case class UserNoMfa(username: String, userLastActiveDay: Option[Long], tags: List[Tag]) extends IAMAlert
 
-case class IAMAlertTargetGroup(targets: List[Target], outdatedKeysUsers: Seq[UserWithOutdatedKeys], noMfaUsers: Seq[UserNoMfa])
\ No newline at end of file
+sealed trait IamAuditNotificationType {def name: String}
+object Warning extends IamAuditNotificationType {val name = "Warning"}
+object Final extends IamAuditNotificationType {val name = "Final"}
+
+case class IamAuditEmail(dateNotificationSent: DateTime, notificationType: IamAuditNotificationType)
+case class IamAuditUser(id: String, awsAccount: String, userName: String, emails: List[IamAuditEmail])
+case class IamNotification(iamUser: IamAuditUser, anghammaradNotification: Notification)
```

Comment:
This new class wraps the Anghammarad Notification class, so that we can pass on information about the Iam User, which is needed for recording in DynamoDB.